gunicorn.conf.py

```diff
@@ -0,0 +1,12 @@
+import multiprocessing
+
+use_bind = "0.0.0.0:8000"
+
+cores = multiprocessing.cpu_count()
```

Comment:
this is not a heavy API, we can just use min 2 cores like this min(multiprocessing.cpu_cout(), 2)